src/HttpClient/FileGetContentsClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Voronkovich\SberbankAcquiring\HttpClient;
+
+/**
+ * Simple HTTP client using file_get_contents() with stream_context_create().
+ *
+ * @author     disjointed
+ */
+class FileGetContentsClient implements HttpClientInterface
+{
+	public function request($uri, $method = 'GET', array $headers = array(), array $data = array())
+	{
+		$response = file_get_contents($uri, false, stream_context_create([
```

Comment:
The file_get_contents function returns FALSE on failure. You should throw an `Voronkovich\SberbankAcquiring\Exception\NetworkException` in this case.